apps/src/templates/instructions.html.ejs

```diff
@@ -5,10 +5,10 @@
   <div class='instructions-markdown<% if (locals.markdownClassicMargins) { -%> classic-margins<% } -%>'><%- locals.renderedMarkdown %></div>
 <% } else { -%>
   <% if (locals.instructions) { -%>
-    <p><%= locals.instructions %></p>
```

Comment:
@tanyaparker only downside to this is that we now include instructions/instructions2 text as html, not just text.  last time i looked i didn't find a super easy way to strip html tags in ruby, but i'm sure there's a way if it's really necessary.